terraform/projects/prom-ec2/paas-production/extra-prometheus-scrape-configs.yml.tpl

```diff
@@ -1,3 +1,27 @@
+- job_name: dcs-federate
+  scheme: https
+  honor_labels: true
+  honor_timestamps: true
+  metrics_path: '/federate'
+  params:
+    "match[]":
+    # fetch everything (via https://stackoverflow.com/a/39253848 )
+    - '{__name__=~".+"}'
+  static_configs:
+  - targets:
+    - dcs-sandbox-prometheus.cloudapps.digital
+    labels: # although maybe these could be external_labels on the
+            # scraped prometheus?
```

Comment:
I think we should only annotate the hostname that the metrics were federated _from_ here, because that's all we know for sure at this end. I think it would be better to annotate space/org/region/app name from the federated end, where it can be injected via the terraform template and is less likely to get out of sync.